From f581399ecf1fa1e6e8116ef4cd04677b424f0ca5 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Carlos=20Aznar=C3=A1n?= <caznaranl@uni.pe>
Date: Sun, 10 Mar 2024 18:00:16 -0500
Subject: [PATCH] Enable debugging

---
 .SRCINFO | 2 +-
 PKGBUILD | 7 ++++---
 2 files changed, 5 insertions(+), 4 deletions(-)

diff --git a/.SRCINFO b/.SRCINFO
index 0ce4a66..ade8e4e 100644
--- a/.SRCINFO
+++ b/.SRCINFO
@@ -1,7 +1,7 @@
 pkgbase = petsc
 	pkgdesc = Portable, extensible toolkit for scientific computation
 	pkgver = 3.20.5
-	pkgrel = 2
+	pkgrel = 3
 	url = https://petsc.org
 	install = petsc.install
 	arch = i686
diff --git a/PKGBUILD b/PKGBUILD
index b442f25..b9976fb 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -3,12 +3,12 @@
 # Contributor: Myles English <myles at rockhead dot biz>
 # Contributor: Lucas H. Gabrielli <heitzmann at gmail dot com>
 pkgver=3.20.5
-pkgrel=2
+pkgrel=3
 pkgname=petsc
-_config=linux-c-opt
+# _config=linux-c-opt
 # if --with-debugging=yes is set then PETSC_ARCH is automatically set to
 #"linux-c-debug" for some things, so the _config should be changed too
-#_config=linux-c-debug
+_config=linux-c-debug
 pkgdesc="Portable, extensible toolkit for scientific computation"
 arch=('i686' 'x86_64')
 url="https://petsc.org"
@@ -51,6 +51,7 @@ build() {
   export PETSC_DIR=${_build_dir}
 
   CONFOPTS="--with-shared-libraries=1 \
+            --with-debugging=1 \
             --with-petsc4py=1 \
             --with-mpi-f90module-visibility=0 \
             --with-bison=0 \
-- 
2.44.0

