From a08cefc674bdb97a2fabe2255ae9cf449288c3df Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Carlos=20Aznar=C3=A1n?= <caznaranl@uni.pe>
Date: Tue, 23 Jan 2024 13:48:25 -0500
Subject: [PATCH] Start debugging

---
 PKGBUILD | 20 ++++++++++++++------
 1 file changed, 14 insertions(+), 6 deletions(-)

diff --git a/PKGBUILD b/PKGBUILD
index 635b928..5a6fddd 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -3,18 +3,18 @@
 # Contributor: Myles English <myles at rockhead dot biz>
 # Contributor: Lucas H. Gabrielli <heitzmann at gmail dot com>
 pkgver=3.20.3
-pkgrel=4
+pkgrel=5
 pkgname=petsc
-_config=linux-c-opt
+# _config=linux-c-opt
 # if --with-debugging=yes is set then PETSC_ARCH is automatically set to
 #"linux-c-debug" for some things, so the _config should be changed too
-#_config=linux-c-debug
+_config=linux-c-debug
 pkgdesc="Portable, extensible toolkit for scientific computation"
 arch=('i686' 'x86_64')
 url="https://petsc.org"
 license=('BSD')
 options=(staticlibs)
-depends=('python-numpy' 'openmpi' 'boost' 'lapack' 'hdf5-openmpi' 'fftw' 'superlu' 'suitesparse')
+depends=('python-numpy' 'openmpi' 'boost' 'lapack' 'hdf5-openmpi' 'fftw' 'superlu' 'suitesparse' 'valgrind')
 makedepends=('gcc' 'gcc-fortran' 'cmake' 'cython')
 provides=('petsc4py')
 optdepends=('trilinos: support for trilinos'
@@ -31,13 +31,19 @@ optdepends=('trilinos: support for trilinos'
 
 install=petsc.install
 source=(http://web.cels.anl.gov/projects/petsc/download/release-snapshots/${pkgname}-${pkgver}.tar.gz
-        test_optdepends.sh)
+        test_optdepends.sh
+        https://raw.githubusercontent.com/carlosal1015/test-petsc-complex/main/patches/runex19_superlu_dist.patch)
 sha512sums=('912538850eeaf6d78c090438f8198963cd787ef7ff3bbc841719b08fc738c7b20b7955850baacca4eada4a97b113492b9111d35afa33918ec52123e2f1a73f9b'
-            '80dfd422ba16e0fe529b65fd195d825095483282a2b202f03f81c507cfe694ce0ec9db606cbd74ac68d206343818a6aec15d32aea8faceddd1cec756409d5932')
+            '80dfd422ba16e0fe529b65fd195d825095483282a2b202f03f81c507cfe694ce0ec9db606cbd74ac68d206343818a6aec15d32aea8faceddd1cec756409d5932'
+            'SKIP')
 
 _install_dir=/opt/petsc/${_config}
 _petsc_arch=arch-${_config}
 
+prepare() {
+  cd ${srcdir}/${pkgname}-${pkgver}
+  patch -p1 -i ../runex19_superlu_dist.patch
+}
 
 build() {
   _build_dir=${srcdir}/${pkgname}-${pkgver}
@@ -47,6 +53,8 @@ build() {
   export PETSC_DIR=${_build_dir}
 
   CONFOPTS="--with-shared-libraries=1 \
+            --with-debugging=1 \
+            --with-valgrind=1 \
             --with-petsc4py=1 \
             --with-mpi-f90module-visibility=0 \
             --with-mpi-dir=/usr \
-- 
2.43.0

