From 60b622effb47ede71a5a37461c9bd9af5b50c7b7 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Carlos=20Aznar=C3=A1n?= <caznaranl@uni.pe>
Date: Tue, 23 Jan 2024 10:27:08 -0500
Subject: [PATCH] Start debugging

---
 PKGBUILD | 17 ++++++++++++-----
 1 file changed, 12 insertions(+), 5 deletions(-)

diff --git a/PKGBUILD b/PKGBUILD
index 1ff0b9b..e793f25 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -3,12 +3,12 @@
 # Contributor: Myles English <myles at rockhead dot biz>
 # Contributor: Lucas H. Gabrielli <heitzmann at gmail dot com>
 pkgver=3.20.3
-pkgrel=3
+pkgrel=4
 pkgname=petsc
-_config=linux-c-opt
+# _config=linux-c-opt
 # if --with-debugging=yes is set then PETSC_ARCH is automatically set to
 #"linux-c-debug" for some things, so the _config should be changed too
-#_config=linux-c-debug
+_config=linux-c-debug
 pkgdesc="Portable, extensible toolkit for scientific computation"
 arch=('i686' 'x86_64')
 url="https://petsc.org"
@@ -32,13 +32,19 @@ optdepends=('trilinos: support for trilinos'
 
 install=petsc.install
 source=(http://web.cels.anl.gov/projects/petsc/download/release-snapshots/${pkgname}-${pkgver}.tar.gz
-        test_optdepends.sh)
+        test_optdepends.sh
+        https://raw.githubusercontent.com/carlosal1015/test-petsc-complex/main/patches/runex19_superlu_dist.patch)
 sha512sums=('912538850eeaf6d78c090438f8198963cd787ef7ff3bbc841719b08fc738c7b20b7955850baacca4eada4a97b113492b9111d35afa33918ec52123e2f1a73f9b'
-            '80dfd422ba16e0fe529b65fd195d825095483282a2b202f03f81c507cfe694ce0ec9db606cbd74ac68d206343818a6aec15d32aea8faceddd1cec756409d5932')
+            '80dfd422ba16e0fe529b65fd195d825095483282a2b202f03f81c507cfe694ce0ec9db606cbd74ac68d206343818a6aec15d32aea8faceddd1cec756409d5932'
+            'SKIP')
 
 _install_dir=/opt/petsc/${_config}
 _petsc_arch=arch-${_config}
 
+prepare() {
+  cd ${srcdir}/${pkgname}-${pkgver}
+  patch -p1 -i ../runex19_superlu_dist.patch
+}
 
 build() {
   _build_dir=${srcdir}/${pkgname}-${pkgver}
@@ -48,6 +54,7 @@ build() {
   export PETSC_DIR=${_build_dir}
 
   CONFOPTS="--with-shared-libraries=1 \
+            --with-debugging=1 \
             --with-petsc4py=1 \
             --with-mpi-f90module-visibility=0 \
             --with-mpi-dir=/usr \
-- 
2.43.0

