diff --color -ruN petsc-3.20.3/src/snes/tutorials/makefile petsc-3.20.3-patched/src/snes/tutorials/makefile
--- petsc-3.20.3/src/snes/tutorials/makefile	2023-11-30 14:14:32.000000000 -0500
+++ petsc-3.20.3-patched/src/snes/tutorials/makefile	2024-01-23 09:45:18.699637765 -0500
@@ -101,8 +101,9 @@
            fi; \
            ${RM} -f ex19_6.tmp
 runex19_superlu_dist:
-	-@${MPIEXEC} -n 1 ${MPIEXEC_TAIL} ./ex19 -da_grid_x 20 -da_grid_y 20 -pc_type lu -pc_factor_mat_solver_type superlu_dist > ex19.tmp 2>&1; \
-	   if (${DIFF} output/ex19_superlu.out ex19.tmp) then \
+	-@${MPIEXEC} -valgrind -n 1 ${MPIEXEC_TAIL} ./ex19 -da_grid_x 20 -da_grid_y 20 -pc_type lu -pc_factor_mat_solver_type superlu_dist > ex19.tmp; \
+	   echo ${MPIEXEC} ${MPIEXEC_TAIL}
+     if (${DIFF} output/ex19_superlu.out ex19.tmp) then \
            echo "C/C++ example src/snes/tutorials/ex19 run successfully with superlu_dist"; \
            else  \
              printf "${PWD}\nPossible problem with ex19 running with superlu_dist, diffs above\n=========================================\n"; \
